pkg/branding/brandedtext_test.go

```diff
@@ -0,0 +1,64 @@
+package branding
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/pkg/testutils/envisolator"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	brandedProductNameRHACS    = "Red Hat Advanced Cluster Security for Kubernetes"
+	brandedProductNameStackrox = "StackRox"
+)
+
+var _ suite.TearDownTestSuite = (*BrandedTextTestSuite)(nil)
+
+func TestBrandedText(t *testing.T) {
+	suite.Run(t, new(BrandedTextTestSuite))
+}
+
+type BrandedTextTestSuite struct {
+	suite.Suite
+	envIsolator *envisolator.EnvIsolator
+}
+
+func (s *BrandedTextTestSuite) SetupSuite() {
+	s.envIsolator = envisolator.NewEnvIsolator(s.T())
+}
+
+func (s *BrandedTextTestSuite) TearDownTest() {
+	s.envIsolator.RestoreAll()
+}
+
+func (s *BrandedTextTestSuite) TestGetBrandedProductName() {
+	tests := map[string]struct {
+		productBrandingEnv string
+		brandedProductName string
+	}{
+		"RHACS branding": {
+			productBrandingEnv: "RHACS_BRANDING",
+			brandedProductName: brandedProductNameRHACS,
+		},
+		"Stackrox branding": {
+			productBrandingEnv: "STACKROX_BRANDING",
+			brandedProductName: brandedProductNameStackrox,
+		},
+		"Default setting": {
+			productBrandingEnv: "ROX_PRODUCT_BRANDING",
+			brandedProductName: brandedProductNameStackrox,
+		},
+		// TODO #ROX-10208: Change this to StackRox after changing the default value of ProductBrandingEnvName.
```

Comment:
We don't use `#` to reference JIRA IDs. The pattern should be 
```suggestion
		// TODO(ROX-10208): Change this to StackRox after changing the default value of ProductBrandingEnvName.
```

Reply:
Done